core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -431,7 +438,22 @@
    * {@link SqlConformanceEnum#STRICT_2003},
    * {@link SqlConformanceEnum#ORACLE_10},
    * {@link SqlConformanceEnum#ORACLE_12};
+   * {@link SqlConformanceEnum#PRESTO};
    * true otherwise.
    */
   boolean allowQualifyingCommonColumn();
+
+  /**
+   * Whether directly alias array columns in UNNEST.
+   *
+   * <p>E.g. in UNNEST(a_array, b_array) AS T(a, b),
+   * a and b will be aliases of a_array and b_array
+   * respectively
```

Comment:
What will be the behavior of that expression if `allowAliasUnnestColumns` is false?

Reply:
What is the most logical place for this function inside SqlConformance? I bet it's not at the very end.